### PR TITLE
Section 5 Updates

### PR DESCRIPTION
**Pull request by lachellel**
@jaspenc

I pulled in your PR #135 - added your changes directly to the certificate-policy markdown
Comments and questions will be inline...

### CODE REVIEW

File: certificate-policy.md [hunk 1]
```diff
@@ -924,40 +924,143 @@ Based on the Risk Assessment, the CA SHALL develop, implement, and maintain a se
...
+Full system backups sufficient to recover from system failure SHALL be made on a periodic schedule.  Backups are to be performed and stored off-site not less than once per week.  At least one full backup copy SHALL be stored at an off-site location (separate from CA equipment).  Only the latest full backup need be retained.  The backup SHALL be stored at a site with physical and procedural controls commensurate to that of the operational CA.
 
 ## 5.2 Procedural controls
 
 ### 5.2.1 Trusted roles
+A trusted role is one whose incumbent performs functions that can introduce security problems if not carried out properly, whether accidentally or maliciously.  
+The requirements of this policy are defined in terms of four roles.   
+
+1.	Administrator – authorized to install, configure, and maintain the CA; establish and maintain system accounts; configure audit parameters; and generate component keys. 
+2.	Officer – authorized to request or approve certificate issuance and revocations.
```
Comment (lachellel) [inline certificate-policy.md]: I think the CA Officer needs to be clarified here and in the other sections?  This does not need to be a human (for one) and can be automated.

File: certificate-policy.md [hunk 2]
```diff
@@ -924,40 +924,143 @@ Based on the Risk Assessment, the CA SHALL develop, implement, and maintain a se
...
 
 ### 5.3.2 Background check procedures
+Trusted role personnel SHALL, at a minimum, pass a background investigation covering the following areas:
+•	Employment;
+•	Education;
+•	Place of residence;
+•	Law Enforcement; and
+•	References.
+The period of investigation must cover at least the last five years for each area, excepting the residence check which must cover at least the last three years.  Regardless of the date of award, the highest educational degree SHALL be verified.
+Adjudication of the background investigation SHALL be performed by a competent adjudication authority using a process consistent with Executive Order 12968 August 1995, or equivalent.
```
Comment (lachellel) [inline certificate-policy.md]: Update to  Executive Order 13467, June 30, 2008?

File: certificate-policy.md [hunk 3]
```diff
@@ -1088,12 +1259,43 @@ In the event of a mis-issuance, the issuing CA SHALL conduct the following actio
...
+-	If the CA signature keys are not destroyed, CA operation SHALL be reestablished, giving priority to the ability to generate certificate status information within the CRL issuance schedule.
+- If the CA signature keys are destroyed, CA operation SHALL be reestablished as quickly as possible, giving priority to the generation of a new CA key pair.
 
 ### 5.7.3 Recovery Procedures after Key Compromise
+In the event of a CA private key compromise, the following operations must be performed. 
+
+- The FPKI Policy Authority SHALL be immediately informed, as well as any superior CAs and any entities known to be distributing the CA certificate.
+-	The CA must generate new keys.
+
+If the CA distributed the private key in a Trusted Certificate, the CA SHALL perform the following operations:
```
Comment (lachellel) [inline certificate-policy.md]: This  is a bit confusing?  Can we clarify?

**Comment by lachellel** [comment]
See comments and questions in-line: https://github.com/uspki/policies/pull/136/files

**Comment by lachellel** [comment]
@konklone 
merge?

**Comment by konklone** [comment]
:+1:

**Pull request merged**